service/src/main/java/org/cbioportal/service/impl/ExpressionEnrichmentServiceImpl.java

```diff
@@ -113,7 +113,7 @@
             }
 
             // ignore gene enrichment if any of the group is ignored
-            if (groupsStatistics.size() == groupIndicesMap.size()) {
+            if (groupsStatistics.size() > 1) {
```

Comment:
@kalletlak Not super familiar with the implementation. Could you elaborate what this does? Maybe good to update the comment as well?

Reply:
@inodb updated comment